components/automate-cli/pkg/verifyserver/server/api/v1/softwareversion.go

```diff
@@ -0,0 +1,24 @@
+package v1
+
+import (
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) CheckSoftwareVersion(c *fiber.Ctx) {
+	if c.Query("node_type") == "" {
```

Comment:
Also, check for node type as well. If the node type is not in our list, we should return the status bad request here.